test/role.test.js

```diff
@@ -172,6 +173,14 @@ describe('role model', function() {
   });
 
   it('should support owner role resolver', function() {
+    Role.registerResolver(Role.EVERYONE_PROMISE, function(role, context) {
+      process.nextTick(function() {
```

Comment:
Also notice that Promise `resolve` function does not have error-first signature, you pass only the result.

```js
// wrong
resolve(null, true);
resolve(err);

// correct
resolve(true);
reject(err);
```